src/Core/Form/IdentifiableObject/DataProvider/CategoryFormDataProvider.php

```diff
@@ -52,26 +52,26 @@ final class CategoryFormDataProvider implements FormDataProviderInterface
     private $contextShopRootCategoryId;
 
     /**
-     * @var DefaultGroupsProviderInterface
+     * @var GroupDataProvider
      */
-    private $defaultGroupsProvider;
+    private $groupDataProvider;
 
     /**
      * @param CommandBusInterface $queryBus
      * @param int $contextShopId
      * @param int $contextShopRootCategoryId
-     * @param DefaultGroupsProviderInterface $defaultGroupsProvider
+     * @param GroupDataProvider $groupDataProvider
      */
     public function __construct(
         CommandBusInterface $queryBus,
         $contextShopId,
         $contextShopRootCategoryId,
-        DefaultGroupsProviderInterface $defaultGroupsProvider
+        GroupDataProvider $groupDataProvider
```

Comment:
@jf-viguier isn't it a BC break if we change type of the required parameter?

Reply:
I don't know, please help. This class is only used in service declaration.

Reply:
I've added this change in the pr description.